.github/workflows/ci.yml

```diff
@@ -1,75 +1,95 @@
-name: Full Service CI
+name: ci
 
 env:
-  DOCKERHUB_REPO: mobilecoin/full-service
+  SGX_MODE: HW
+  IAS_MODE: PROD
```

Comment:
I notice the old config used SW/DEV. Is HW/PROD the right combo for CI? I'd guess IAS_MODE=DEV would be better.